backend/fhtways/graph_setup_coord.py

```diff
@@ -56,134 +56,175 @@ def create_graph():
+    G.add_edge("door_left", "stairs_middle", weight=8, description="Gehen Sie vom linken Eingang zum rechten Eingang. Es sind 14 Schritte geradeaus. Achten Sie auf die Veränderungen im Echo des Raums.")
+    G.add_edge("stairs_middle", "door_left", weight=8, description="Gehen Sie vom linken Eingang zum rechten Eingang. Es sind 14 Schritte geradeaus. Achten Sie auf die Veränderungen im Echo des Raums.")
 
-    G.add_edge("door_left", "door_right", weight=14, description="Door left to Door right.")
-    G.add_edge("door_right", "door_left", weight=14, description="Door right to Door left.")
 
-    G.add_edge("door_left", "lift", weight=10, description="Door left to Lift.")
-    G.add_edge("lift", "door_left", weight=10, description="Lift to Door left.")
+    G.add_edge("door_left", "door_right", weight=14, description="Gehen Sie 14 Schritte geradeaus, um die Tür vor Ihnen zu erreichen. Achten Sie auf die Beschaffenheit des Bodens und die Akustik der Umgebung, um Ihre Bewegung zu orientieren.")
+    G.add_edge("door_right", "door_left", weight=14, description="Gehen Sie 14 Schritte geradeaus, um die Tür vor Ihnen zu erreichen. Achten Sie auf die Beschaffenheit des Bodens und die Akustik der Umgebung, um Ihre Bewegung zu orientieren.")
 
-    G.add_edge("door_left", "toilets", weight=3, description="Biegen Sie gleich nach rechts ab und machen Sie 3 Schritte. Öffnen Sie die Tür und Sie erreichen die Toilette.")
-    G.add_edge("toilets", "door_left", weight=3, description="Öffnen Sie die Tür der Toilette, machen Sie 3 Schritte geradaus, biegen Sie nach links ab. Öffnen Sie die Tür vor Ihnen und gehen Sie geradeaus.")
-    
-    G.add_edge("toilets", "stairs-middle", weight=6, description="Öffnen Sie die Toilettentür, machen Sie 2 Schritte geradaus, biegen Sie nach rechts ab und machen Sie 2 Schritte geradeaus. Biegen Sie nach links ab und Sie erreichen die Stiegen des 4.Stocks.")
-    G.add_edge("stairs-middle", "toilets", weight=6, description="Nachdem Sie den 4.Stock erreicht haben, gehen Sie geradeaus, indem Sie 2 Schritte machen und nach recht abbiegen. Danach machen Sie 2 Schritte geradeaus und biegen Sie demnächst nach links ab. Öffnen Sie die Tür vor Ihnen und Sie erreichen die Toilette.")
 
-    G.add_edge("toilets", "lift", weight=4, description="Öffnen Sie die Toilettentür, machen Sie 2 Schritte geradaus, biegen Sie nach rechts ab und machen Sie 2 Schritte geradeaus. Biegen Sie nach rechts ab und Sie erreichen die Lifte.")
-    G.add_edge("lift", "toilets", weight=4, description="Verlassen Sie den Lift, indem Sie 2 Schritte geradeaus machen und nach links abbiegen. Danach machen Sie 2 Schritte geradeaus und biegen Sie demnächst nach links ab. Öffnen Sie die Tür vor Ihnen und Sie erreichen die Toilette.")
+    G.add_edge("door_left", "lift", weight=10, description="Gehen Sie 10 Schritte geradeaus, während Sie sich auf der rechten Seite halten. Fühlen Sie die Wand und orientieren Sie sich in der Umgebung, um den Weg zum Lift zu finden.")
```

Comment:
this is too vague. "orientieren Sie sich in der Umgebung" - we are the ones who need to provide that "Umgebung". we need to explicitly describe what exactly the person should do and where to go, and not just tell them that they should orient themselves on something they are not familiar with.

Reply:
Done, like this:
G.add_edge("door_left", "lift", weight=10, description="Nachdem Sie zur Tür gelangt haben, die zur Halle der 4.Etage führt, gehen Sie zur rechten Seite des Raumes, um die rechte Wand zu erreichen. Von dort aus gehen Sie 10 Schritte geradeaus entlang der rechten Wand. Wenn Sie die Lifttaste erreichen, die zirka auf Ihrer Brusthöhe befindet, drücken Sie sie, um den Aufzug zu rufen.")

Reply:
Looks good